test/features/discord.feature

```diff
@@ -51,3 +51,14 @@ Feature: discord module
     When make a DELETE request to "/discord/123"
     Then the response status code should be 200
     And the response should be "{}"
+
+  Scenario: delete non-existing user
```

Comment:
Added Test for deleting non-existent user

Reply:
Great! thank you 👍